.perltidyrc

```diff
@@ -0,0 +1,15 @@
+-pbp               # Start with Perl Best Practices
+--maximum-line-length=120
+--maximum-consecutive-blank-lines=2
+--indent-columns=2
```

Comment:
Could we do a four-column indent? That's what @nelson-ferraz started with and I don't want to break that convention.